### PR TITLE
Replace caigo with starknet.go

### PR DESCRIPTION
**Pull request by rianhughes**
Renaming the project from caigo to starknet.go has created issues with importing the starknet.go package. Here I simply find-replace "caigo" with "starknet.go"

### CODE REVIEW

File: examples/account/main.go [hunk 1]
```diff
@@ -35,10 +35,10 @@ func main() {
 	fmt.Println("Counter is currently at: ", callResp[0])
 
 	// init account handler
-	ks := caigo.NewMemKeystore()
+	ks := starknet.go.NewMemKeystore()
```
Comment (cicr99) [inline examples/account/main.go]: Wouldn't this cause conflict? Also package name in previous files is `starknetgo`
This repeats other times as well

Comment (rianhughes) [reply examples/account/main.go]: Yes this is an error. I thought I caught all of these! Thanks, I'll have another look now

Comment (rianhughes) [reply examples/account/main.go]: Ah wait, I didn't catch this because I was neglecting broken tests and examples. I'll fix the examples

Comment (rianhughes) [reply examples/account/main.go]: I fixed the import errors in examples, but it's worth noting that they are still broken and should be fixed in following PRs

**Review by cicr99 — APPROVED** [review]
lgtm

**Pull request merged**